static/menu.js

```diff
@@ -0,0 +1,6 @@
+let menu = document.getElementById("collapsible-menu");
+menu.classList = "collapse navbar-collapse";
+
+let button = document.querySelector("nav .navbar-toggler");
+button.toggleAttribute("hidden");
```

Comment:
Why “toggle”? Do we ever want this to enable the attribute? Otherwise I’d prefer `.hidden = false`.